aptos-move/framework/aptos-framework/sources/object.spec.move

```diff
@@ -2,6 +2,13 @@ spec aptos_framework::object {
 
     spec module {
         pragma aborts_if_is_strict;
+        apply AddressNoChange to * except delete;
+    }
+
+    spec schema AddressNoChange {
+        // property 4: Objects may never change the address which houses them.
```

Comment:
@junkil-park Could you double check this property?

Reply:
it is useless because it's implied by property 1, it should be deleted